ci/OWNERS

```diff
@@ -15,9 +15,9 @@
 
 # CI
 /.github/*_TEMPLATE*                    @SigmaSquadron
-/.github/actions                        @NixOS/Security @Mic92 @zowoq @infinisil @azuwis @wolfgangwalther @philiptaron
-/.github/workflows                      @NixOS/Security @Mic92 @zowoq @infinisil @azuwis @wolfgangwalther @philiptaron
-/ci                                     @NixOS/Security @Mic92 @zowoq @infinisil @azuwis @wolfgangwalther @philiptaron
+/.github/actions                        @NixOS/Security @NixOS/Nixpkgs-CI
+/.github/workflows                      @NixOS/Security @NixOS/Nixpkgs-CI
+/ci                                     @NixOS/Security @NixOS/Nixpkgs-CI
```

Comment:
Please drop security. The number of notifications is just too much, so we need to put more trust into team members, which is fine by me.

cc @NixOS/security

Reply:
Fine by me too especially now that we have people working on it and not only drive-by PRs